src/components.js

```diff
@@ -125,23 +125,34 @@ export class Diagram extends Component {
     render() {
         let options = this.props.options == null ? '' : `[${this.props.options}]`
 
+        let cells = this.toArray().map(entries => entries.map(entry =>
+            entry == null ? ''
+            : [
+                (() => {
+                    let value = entry.node.props.value;
+                    let [w1, w2] = value != null
+                        && needWrapChars.some(c => value.includes(c))
+                        ? ['{', '}'] : ['', '']
+
+                    return value != null ? `${w1}${value}${w2}` : '{}'
+                })(),
+                ...entry.edges.map(e => renderEdge(e, this.props.co))
+            ].join(' ')
+        ))
+
+        if(this.props.align && cells.length > 0) {
```

Comment:
To make the code style consistent, please leave a space after `if` and `for` and before `(`.